python/triton/language/extra/hip/libdevice.py

```diff
@@ -0,0 +1,433 @@
+from triton.language import core
+
+
+@core.extern
+def abs(arg0, _builder=None):
+    return core.extern_elementwise(
+        "", "", [arg0], {
+            (core.dtype("int32"), ): ("__ocml_trition_abs", core.dtype("int32")),
```

Comment:
I know I suggested to use `__ocml_triton` prefix erlier. But thinking it a bit, it's better to use `__triton_hip` as the prefix instead--typically we want the broader scope context earlier to avoid potential symbol collison. Sorry about the back and forth. (BTW you have a typo here--it should be triton.)

Reply:
Honestly, `__triton_hip` makes much more sense for me. It feels like it is some special case and, thus, one can expect to have a dedicate function/method for converting such instructions (which we have).

Done